2016/09/20 20:33:49 [error] 4090#0: [lua] orange.lua:159: init(): Startup error: ./orange/utils/utils.lua:170: ./orange/utils/handle.lua:5: module 'resty.template' not found:
	no field package.preload['resty.template']
	no file '../resty/template.lua'
	no file '/usr/local/lor/resty/template.lua'
	no file '/opt/openresty/lualib/resty/template.lua'
	no file '/opt/openresty/lualib/resty/template/init.lua'
	no file './resty/template.lua'
	no file '/opt/openresty/luajit/share/luajit-2.1.0-beta1/resty/template.lua'
	no file '/usr/local/share/lua/5.1/resty/template.lua'
	no file '/usr/local/share/lua/5.1/resty/template/init.lua'
	no file '/opt/openresty/luajit/share/lua/5.1/resty/template.lua'
	no file '/opt/openresty/luajit/share/lua/5.1/resty/template/init.lua'
	no file '/opt/openresty/lualib/resty/template.so'
	no file './resty/template.so'
	no file '/usr/local/lib/lua/5.1/resty/template.so'
	no file '/opt/openresty/luajit/lib/lua/5.1/resty/template.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
	no file '/opt/openresty/lualib/resty.so'
	no file './resty.so'
	no file '/usr/local/lib/lua/5.1/resty.so'
	no file '/opt/openresty/luajit/lib/lua/5.1/resty.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
2016/09/20 21:11:59 [error] 4559#0: [lua] orange.lua:159: init(): Startup error: ./orange/utils/utils.lua:170: ./orange/utils/handle.lua:5: module 'resty.template' not found:
	no field package.preload['resty.template']
	no file '../resty/template.lua'
	no file './lor/resty/template.lua'
	no file '/opt/openresty/lualib/resty/template.lua'
	no file '/opt/openresty/lualib/resty/template/init.lua'
	no file './resty/template.lua'
	no file '/opt/openresty/luajit/share/luajit-2.1.0-beta1/resty/template.lua'
	no file '/usr/local/share/lua/5.1/resty/template.lua'
	no file '/usr/local/share/lua/5.1/resty/template/init.lua'
	no file '/opt/openresty/luajit/share/lua/5.1/resty/template.lua'
	no file '/opt/openresty/luajit/share/lua/5.1/resty/template/init.lua'
	no file '/opt/openresty/lualib/resty/template.so'
	no file './resty/template.so'
	no file '/usr/local/lib/lua/5.1/resty/template.so'
	no file '/opt/openresty/luajit/lib/lua/5.1/resty/template.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
	no file '/opt/openresty/lualib/resty.so'
	no file './resty.so'
	no file '/usr/local/lib/lua/5.1/resty.so'
	no file '/opt/openresty/luajit/lib/lua/5.1/resty.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
2016/09/20 21:13:07 [error] 4701#0: [lua] orange.lua:159: init(): Startup error: ./orange/utils/utils.lua:170: ./orange/utils/handle.lua:5: module 'resty.template' not found:
	no field package.preload['resty.template']
	no file '../resty/template.lua'
	no file '../lor/resty/template.lua'
	no file '/opt/openresty/lualib/resty/template.lua'
	no file '/opt/openresty/lualib/resty/template/init.lua'
	no file './resty/template.lua'
	no file '/opt/openresty/luajit/share/luajit-2.1.0-beta1/resty/template.lua'
	no file '/usr/local/share/lua/5.1/resty/template.lua'
	no file '/usr/local/share/lua/5.1/resty/template/init.lua'
	no file '/opt/openresty/luajit/share/lua/5.1/resty/template.lua'
	no file '/opt/openresty/luajit/share/lua/5.1/resty/template/init.lua'
	no file '/opt/openresty/lualib/resty/template.so'
	no file './resty/template.so'
	no file '/usr/local/lib/lua/5.1/resty/template.so'
	no file '/opt/openresty/luajit/lib/lua/5.1/resty/template.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
	no file '/opt/openresty/lualib/resty.so'
	no file './resty.so'
	no file '/usr/local/lib/lua/5.1/resty.so'
	no file '/opt/openresty/luajit/lib/lua/5.1/resty.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
2016/09/20 21:14:01 [emerg] 4774#0: bind() to 0.0.0.0:80 failed (13: Permission denied)
ngx_http_fastdfs_process_init pid=4846
ngx_http_fastdfs_process_init pid=4849
ngx_http_fastdfs_process_init pid=4848
ngx_http_fastdfs_process_init pid=4850
[2016-09-20 21:15:21] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-20 21:15:21] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
2016/09/20 21:15:21 [error] 4846#0: [lua] mysql_db.lua:31: query(): failed to connect: Access denied for user 'root'@'localhost' (using password: NO): 1045 28000, context: ngx.timer
2016/09/20 21:15:21 [error] 4846#0: [lua] mysql_db.lua:31: query(): failed to connect: Access denied for user 'root'@'localhost' (using password: NO): 1045 28000, context: ngx.timer
2016/09/20 21:15:21 [error] 4846#0: [lua] mysql_db.lua:31: query(): failed to connect: Access denied for user 'root'@'localhost' (using password: NO): 1045 28000, context: ngx.timer
2016/09/20 21:15:21 [error] 4846#0: [lua] mysql_db.lua:31: query(): failed to connect: Access denied for user 'root'@'localhost' (using password: NO): 1045 28000, context: ngx.timer
2016/09/20 21:15:21 [error] 4846#0: [lua] mysql_db.lua:31: query(): failed to connect: Access denied for user 'root'@'localhost' (using password: NO): 1045 28000, context: ngx.timer
2016/09/20 21:15:21 [error] 4846#0: [lua] mysql_db.lua:31: query(): failed to connect: Access denied for user 'root'@'localhost' (using password: NO): 1045 28000, context: ngx.timer
2016/09/20 21:15:21 [error] 4846#0: [lua] mysql_db.lua:31: query(): failed to connect: Access denied for user 'root'@'localhost' (using password: NO): 1045 28000, context: ngx.timer
2016/09/20 21:15:21 [error] 4846#0: [lua] mysql_db.lua:31: query(): failed to connect: Access denied for user 'root'@'localhost' (using password: NO): 1045 28000, context: ngx.timer
[2016-09-20 21:15:21] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-20 21:15:21] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-20 21:15:21] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-20 21:15:21] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-20 21:15:21] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-20 21:15:21] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
2016/09/20 21:16:28 [info] 4846#0: *10 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/21 15:20:24 [notice] 3641#0: signal process started
2016/09/21 15:20:24 [alert] 3641#0: kill(4845, 15) failed (3: No such process)
2016/09/21 15:20:24 [error] 3644#0: [lua] orange.lua:206: init(): Startup error: ./orange/utils/config_loader.lua:22: Expected object key string but found T_OBJ_END at character 828
2016/09/21 15:25:23 [notice] 3712#0: signal process started
2016/09/21 15:25:23 [alert] 3712#0: kill(4845, 15) failed (3: No such process)
2016/09/21 15:25:23 [error] 3715#0: [lua] orange.lua:206: init(): Startup error: module 'libs.pgmoon.init' not found:
	no field package.preload['libs.pgmoon.init']
	no file '../libs/pgmoon/init.lua'
	no file './lor/lor/libs/pgmoon/init.lua'
	no file '/opt/openresty/lualib/libs/pgmoon/init.lua'
	no file '/opt/openresty/lualib/libs/pgmoon/init/init.lua'
	no file './libs/pgmoon/init.lua'
	no file '/opt/openresty/luajit/share/luajit-2.1.0-beta1/libs/pgmoon/init.lua'
	no file '/usr/local/share/lua/5.1/libs/pgmoon/init.lua'
	no file '/usr/local/share/lua/5.1/libs/pgmoon/init/init.lua'
	no file '/opt/openresty/luajit/share/lua/5.1/libs/pgmoon/init.lua'
	no file '/opt/openresty/luajit/share/lua/5.1/libs/pgmoon/init/init.lua'
	no file '/opt/openresty/lualib/libs/pgmoon/init.so'
	no file './libs/pgmoon/init.so'
	no file '/usr/local/lib/lua/5.1/libs/pgmoon/init.so'
	no file '/opt/openresty/luajit/lib/lua/5.1/libs/pgmoon/init.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
	no file '/opt/openresty/lualib/libs.so'
	no file './libs.so'
	no file '/usr/local/lib/lua/5.1/libs.so'
	no file '/opt/openresty/luajit/lib/lua/5.1/libs.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
2016/09/21 15:28:19 [notice] 3752#0: signal process started
2016/09/21 15:28:19 [alert] 3752#0: kill(4845, 15) failed (3: No such process)
2016/09/21 15:28:19 [error] 3755#0: [lua] orange.lua:206: init(): Startup error: module 'pgmoon.init' not found:
	no field package.preload['pgmoon.init']
	no file '../pgmoon/init.lua'
	no file './lor/lor/pgmoon/init.lua'
	no file '/opt/openresty/lualib/pgmoon/init.lua'
	no file '/opt/openresty/lualib/pgmoon/init/init.lua'
	no file './pgmoon/init.lua'
	no file '/opt/openresty/luajit/share/luajit-2.1.0-beta1/pgmoon/init.lua'
	no file '/usr/local/share/lua/5.1/pgmoon/init.lua'
	no file '/usr/local/share/lua/5.1/pgmoon/init/init.lua'
	no file '/opt/openresty/luajit/share/lua/5.1/pgmoon/init.lua'
	no file '/opt/openresty/luajit/share/lua/5.1/pgmoon/init/init.lua'
	no file '/opt/openresty/lualib/pgmoon/init.so'
	no file './pgmoon/init.so'
	no file '/usr/local/lib/lua/5.1/pgmoon/init.so'
	no file '/opt/openresty/luajit/lib/lua/5.1/pgmoon/init.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
	no file '/opt/openresty/lualib/pgmoon.so'
	no file './pgmoon.so'
	no file '/usr/local/lib/lua/5.1/pgmoon.so'
	no file '/opt/openresty/luajit/lib/lua/5.1/pgmoon.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
2016/09/21 15:28:28 [notice] 3774#0: signal process started
2016/09/21 15:28:28 [alert] 3774#0: kill(4845, 15) failed (3: No such process)
2016/09/21 15:28:28 [error] 3777#0: [lua] orange.lua:206: init(): Startup error: ./lor/lor/resty/pgmoon/init.lua:1: module 'pgmoon.socket' not found:
	no field package.preload['pgmoon.socket']
	no file '../pgmoon/socket.lua'
	no file './lor/lor/pgmoon/socket.lua'
	no file '/opt/openresty/lualib/pgmoon/socket.lua'
	no file '/opt/openresty/lualib/pgmoon/socket/init.lua'
	no file './pgmoon/socket.lua'
	no file '/opt/openresty/luajit/share/luajit-2.1.0-beta1/pgmoon/socket.lua'
	no file '/usr/local/share/lua/5.1/pgmoon/socket.lua'
	no file '/usr/local/share/lua/5.1/pgmoon/socket/init.lua'
	no file '/opt/openresty/luajit/share/lua/5.1/pgmoon/socket.lua'
	no file '/opt/openresty/luajit/share/lua/5.1/pgmoon/socket/init.lua'
	no file '/opt/openresty/lualib/pgmoon/socket.so'
	no file './pgmoon/socket.so'
	no file '/usr/local/lib/lua/5.1/pgmoon/socket.so'
	no file '/opt/openresty/luajit/lib/lua/5.1/pgmoon/socket.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
	no file '/opt/openresty/lualib/pgmoon.so'
	no file './pgmoon.so'
	no file '/usr/local/lib/lua/5.1/pgmoon.so'
	no file '/opt/openresty/luajit/lib/lua/5.1/pgmoon.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
2016/09/21 15:31:07 [notice] 3846#0: signal process started
2016/09/21 15:31:07 [alert] 3846#0: kill(4845, 15) failed (3: No such process)
2016/09/21 15:31:07 [error] 3849#0: [lua] orange.lua:206: init(): Startup error: ./orange/store/pgsql_store.lua:10: attempt to index local 'options' (a nil value)
2016/09/21 15:32:46 [notice] 3873#0: signal process started
2016/09/21 15:32:46 [alert] 3873#0: kill(4845, 15) failed (3: No such process)
ngx_http_fastdfs_process_init pid=3880
ngx_http_fastdfs_process_init pid=3879
ngx_http_fastdfs_process_init pid=3881
ngx_http_fastdfs_process_init pid=3878
[2016-09-21 15:32:55] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 15:32:55] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 15:32:55] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 15:32:55] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 15:32:55] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 15:32:55] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 15:32:55] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 15:32:55] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
2016/09/21 15:32:55 [error] 3878#0: lua entry thread aborted: runtime error: ./orange/store/pgsql_db.lua:27: attempt to call method 'set_timeout' (a nil value)
stack traceback:
coroutine 0:
	./orange/store/pgsql_db.lua: in function 'query'
	./orange/store/pgsql_store.lua:41: in function 'query'
	./orange/orange.lua:140: in function 'load_data_by_pgsql'
	./orange/orange.lua:229: in function <./orange/orange.lua:223>, context: ngx.timer
2016/09/21 15:35:43 [notice] 3957#0: signal process started
ngx_http_fastdfs_process_exit pid=3878
ngx_http_fastdfs_process_exit pid=3879
ngx_http_fastdfs_process_exit pid=3880
ngx_http_fastdfs_process_exit pid=3881
ngx_http_fastdfs_process_init pid=3965
ngx_http_fastdfs_process_init pid=3964
ngx_http_fastdfs_process_init pid=3963
ngx_http_fastdfs_process_init pid=3962
[2016-09-21 15:35:52] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 15:35:52] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 15:35:52] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 15:35:52] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 15:35:52] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 15:35:52] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 15:35:52] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 15:35:52] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
2016/09/21 15:35:52 [error] 3962#0: lua entry thread aborted: runtime error: ./lor/lor/resty/pgmoon/init.lua:506: missing database for connect
stack traceback:
coroutine 0:
	[C]: in function 'assert'
	./lor/lor/resty/pgmoon/init.lua:506: in function 'send_startup_message'
	./lor/lor/resty/pgmoon/init.lua:169: in function 'connect'
	./orange/store/pgsql_db.lua:29: in function 'query'
	./orange/store/pgsql_store.lua:41: in function 'query'
	./orange/orange.lua:140: in function 'load_data_by_pgsql'
	./orange/orange.lua:229: in function <./orange/orange.lua:223>, context: ngx.timer
2016/09/21 15:41:32 [notice] 4046#0: signal process started
ngx_http_fastdfs_process_exit pid=3962
ngx_http_fastdfs_process_exit pid=3963
ngx_http_fastdfs_process_exit pid=3964
ngx_http_fastdfs_process_exit pid=3965
ngx_http_fastdfs_process_init pid=4051
ngx_http_fastdfs_process_init pid=4055
ngx_http_fastdfs_process_init pid=4054
ngx_http_fastdfs_process_init pid=4053
[2016-09-21 15:41:41] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 15:41:41] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 15:41:41] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 15:41:41] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 15:41:41] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 15:41:41] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
2016/09/21 15:41:41 [error] 4051#0: lua entry thread aborted: runtime error: ./lor/lor/resty/pgmoon/init.lua:506: missing database for connect
stack traceback:
coroutine 0:
	[C]: in function 'assert'
	./lor/lor/resty/pgmoon/init.lua:506: in function 'send_startup_message'
	./lor/lor/resty/pgmoon/init.lua:169: in function 'connect'
	./orange/store/pgsql_db.lua:29: in function 'query'
	./orange/store/pgsql_store.lua:41: in function 'query'
	./orange/orange.lua:140: in function 'load_data_by_pgsql'
	./orange/orange.lua:229: in function <./orange/orange.lua:223>, context: ngx.timer
[2016-09-21 15:41:41] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 15:41:41] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
2016/09/21 15:46:43 [notice] 4150#0: signal process started
ngx_http_fastdfs_process_exit pid=4051
ngx_http_fastdfs_process_exit pid=4053
ngx_http_fastdfs_process_exit pid=4054
ngx_http_fastdfs_process_exit pid=4055
ngx_http_fastdfs_process_init pid=4155
ngx_http_fastdfs_process_init pid=4158
ngx_http_fastdfs_process_init pid=4159
ngx_http_fastdfs_process_init pid=4157
[2016-09-21 15:46:52] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 15:46:52] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 15:46:52] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 15:46:52] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 15:46:52] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 15:46:52] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 15:46:52] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 15:46:52] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
2016/09/21 15:46:52 [error] 4155#0: [lua] pgsql_db.lua:23: query(): key: host value: 127.0.0.1
key: database value: orange
key: max_packet_size value: 1048576
key: user value: fengz
key: password value: zhangfeng
key: port value: 5432
, context: ngx.timer
2016/09/21 15:46:52 [error] 4155#0: lua entry thread aborted: runtime error: ./lor/lor/resty/pgmoon/init.lua:506: missing database for connect
stack traceback:
coroutine 0:
	[C]: in function 'assert'
	./lor/lor/resty/pgmoon/init.lua:506: in function 'send_startup_message'
	./lor/lor/resty/pgmoon/init.lua:169: in function 'connect'
	./orange/store/pgsql_db.lua:31: in function 'query'
	./orange/store/pgsql_store.lua:41: in function 'query'
	./orange/orange.lua:140: in function 'load_data_by_pgsql'
	./orange/orange.lua:229: in function <./orange/orange.lua:223>, context: ngx.timer
2016/09/21 15:55:30 [notice] 4325#0: signal process started
ngx_http_fastdfs_process_exit pid=4155
ngx_http_fastdfs_process_exit pid=4157
ngx_http_fastdfs_process_exit pid=4158
ngx_http_fastdfs_process_exit pid=4159
ngx_http_fastdfs_process_init pid=4333
ngx_http_fastdfs_process_init pid=4332
ngx_http_fastdfs_process_init pid=4331
ngx_http_fastdfs_process_init pid=4330
[2016-09-21 15:55:39] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 15:55:39] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 15:55:39] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 15:55:39] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 15:55:39] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 15:55:39] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 15:55:39] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 15:55:39] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
2016/09/21 15:55:39 [error] 4330#0: lua entry thread aborted: runtime error: ./orange/store/pgsql_db.lua:45: attempt to call method 'set_keepalive' (a nil value)
stack traceback:
coroutine 0:
	./orange/store/pgsql_db.lua: in function 'query'
	./orange/store/pgsql_store.lua:41: in function 'query'
	./orange/orange.lua:140: in function 'load_data_by_pgsql'
	./orange/orange.lua:229: in function <./orange/orange.lua:223>, context: ngx.timer
2016/09/21 15:57:02 [notice] 4417#0: signal process started
ngx_http_fastdfs_process_exit pid=4330
ngx_http_fastdfs_process_exit pid=4331
ngx_http_fastdfs_process_exit pid=4332
ngx_http_fastdfs_process_exit pid=4333
ngx_http_fastdfs_process_init pid=4422
ngx_http_fastdfs_process_init pid=4426
ngx_http_fastdfs_process_init pid=4425
ngx_http_fastdfs_process_init pid=4424
[2016-09-21 15:57:11] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 15:57:11] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 15:57:11] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 15:57:11] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 15:57:11] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 15:57:11] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 15:57:11] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 15:57:11] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
2016/09/21 15:57:11 [error] 4422#0: [lua] pgsql_store.lua:43: query(): PGSQLStore:query, error:1 sql:select key, value from meta where key like '%.enable', context: ngx.timer
2016/09/21 15:57:11 [error] 4422#0: [lua] pgsql_store.lua:43: query(): PGSQLStore:query, error:1 sql:select value from monitor order by id asc, context: ngx.timer
2016/09/21 15:57:11 [error] 4422#0: [lua] pgsql_store.lua:43: query(): PGSQLStore:query, error:1 sql:select value from redirect order by id asc, context: ngx.timer
2016/09/21 15:57:11 [error] 4422#0: [lua] pgsql_store.lua:43: query(): PGSQLStore:query, error:1 sql:select value from rewrite order by id asc, context: ngx.timer
2016/09/21 15:57:11 [error] 4422#0: [lua] pgsql_store.lua:43: query(): PGSQLStore:query, error:1 sql:select value from basic_auth order by id asc, context: ngx.timer
2016/09/21 15:57:11 [error] 4422#0: [lua] pgsql_store.lua:43: query(): PGSQLStore:query, error:1 sql:select value from key_auth order by id asc, context: ngx.timer
2016/09/21 15:57:11 [error] 4422#0: [lua] pgsql_store.lua:43: query(): PGSQLStore:query, error:1 sql:select value from waf order by id asc, context: ngx.timer
2016/09/21 15:57:11 [error] 4422#0: [lua] pgsql_store.lua:43: query(): PGSQLStore:query, error:1 sql:select value from divide order by id asc, context: ngx.timer
2016/09/21 16:00:36 [notice] 4519#0: signal process started
ngx_http_fastdfs_process_exit pid=4422
ngx_http_fastdfs_process_exit pid=4424
ngx_http_fastdfs_process_exit pid=4425
ngx_http_fastdfs_process_exit pid=4426
ngx_http_fastdfs_process_init pid=4524
ngx_http_fastdfs_process_init pid=4528
ngx_http_fastdfs_process_init pid=4527
ngx_http_fastdfs_process_init pid=4525
[2016-09-21 16:00:45] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 16:00:45] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 16:00:45] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 16:00:45] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 16:00:45] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 16:00:45] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 16:00:45] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 16:00:45] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
2016/09/21 16:04:32 [notice] 4703#0: signal process started
ngx_http_fastdfs_process_exit pid=4524
ngx_http_fastdfs_process_exit pid=4525
ngx_http_fastdfs_process_exit pid=4527
ngx_http_fastdfs_process_exit pid=4528
ngx_http_fastdfs_process_init pid=4711
ngx_http_fastdfs_process_init pid=4710
ngx_http_fastdfs_process_init pid=4709
ngx_http_fastdfs_process_init pid=4708
[2016-09-21 16:04:41] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 16:04:41] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 16:04:41] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 16:04:41] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 16:04:41] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 16:04:41] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 16:04:41] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 16:04:41] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
2016/09/21 16:54:41 [notice] 5073#0: signal process started
ngx_http_fastdfs_process_exit pid=4708
ngx_http_fastdfs_process_exit pid=4711
ngx_http_fastdfs_process_exit pid=4709
ngx_http_fastdfs_process_exit pid=4710
ngx_http_fastdfs_process_init pid=5082
ngx_http_fastdfs_process_init pid=5080
ngx_http_fastdfs_process_init pid=5079
ngx_http_fastdfs_process_init pid=5078
[2016-09-21 16:54:50] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 16:54:50] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 16:54:50] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 16:54:50] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 16:54:50] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 16:54:50] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
2016/09/21 16:54:41 [error] 5080#0: init_worker_by_lua error: ./orange/plugins/monitor/api.lua:10: module 'orange.plugins.monitor_db' not found:
	no field package.preload['orange.plugins.monitor_db']
	no file '../orange/plugins/monitor_db.lua'
	no file './lor/lor/orange/plugins/monitor_db.lua'
	no file './lor/lor/resty/orange/plugins/monitor_db.lua'
	no file '/opt/openresty/lualib/orange/plugins/monitor_db.lua'
	no file '/opt/openresty/lualib/orange/plugins/monitor_db/init.lua'
	no file './orange/plugins/monitor_db.lua'
	no file '/opt/openresty/luajit/share/luajit-2.1.0-beta1/orange/plugins/monitor_db.lua'
	no file '/usr/local/share/lua/5.1/orange/plugins/monitor_db.lua'
	no file '/usr/local/share/lua/5.1/orange/plugins/monitor_db/init.lua'
	no file '/opt/openresty/luajit/share/lua/5.1/orange/plugins/monitor_db.lua'
	no file '/opt/openresty/luajit/share/lua/5.1/orange/plugins/monitor_db/init.lua'
	no file '/opt/openresty/lualib/orange/plugins/monitor_db.so'
	no file './orange/plugins/monitor_db.so'
	no file '/usr/local/lib/lua/5.1/orange/plugins/monitor_db.so'
	no file '/opt/openresty/luajit/lib/lua/5.1/orange/plugins/monitor_db.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
	no file '/opt/openresty/lualib/orange.so'
	no file './orange.so'
	no file '/usr/local/lib/lua/5.1/orange.so'
	no file '/opt/openresty/luajit/lib/lua/5.1/orange.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
stack traceback:
	[C]: in function 'require'
	./orange/plugins/monitor/api.lua:10: in main chunk
	[C]: in function 'require'
	./orange/orange.lua:220: in function 'init_worker'
	init_worker_by_lua:3: in main chunk
[2016-09-21 16:54:50] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 16:54:50] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
2016/09/21 16:54:41 [error] 5078#0: init_worker_by_lua error: ./orange/plugins/monitor/api.lua:10: module 'orange.plugins.monitor_db' not found:
	no field package.preload['orange.plugins.monitor_db']
	no file '../orange/plugins/monitor_db.lua'
	no file './lor/lor/orange/plugins/monitor_db.lua'
	no file './lor/lor/resty/orange/plugins/monitor_db.lua'
	no file '/opt/openresty/lualib/orange/plugins/monitor_db.lua'
	no file '/opt/openresty/lualib/orange/plugins/monitor_db/init.lua'
	no file './orange/plugins/monitor_db.lua'
	no file '/opt/openresty/luajit/share/luajit-2.1.0-beta1/orange/plugins/monitor_db.lua'
	no file '/usr/local/share/lua/5.1/orange/plugins/monitor_db.lua'
	no file '/usr/local/share/lua/5.1/orange/plugins/monitor_db/init.lua'
	no file '/opt/openresty/luajit/share/lua/5.1/orange/plugins/monitor_db.lua'
	no file '/opt/openresty/luajit/share/lua/5.1/orange/plugins/monitor_db/init.lua'
	no file '/opt/openresty/lualib/orange/plugins/monitor_db.so'
	no file './orange/plugins/monitor_db.so'
	no file '/usr/local/lib/lua/5.1/orange/plugins/monitor_db.so'
	no file '/opt/openresty/luajit/lib/lua/5.1/orange/plugins/monitor_db.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
	no file '/opt/openresty/lualib/orange.so'
	no file './orange.so'
	no file '/usr/local/lib/lua/5.1/orange.so'
	no file '/opt/openresty/luajit/lib/lua/5.1/orange.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
stack traceback:
	[C]: in function 'require'
	./orange/plugins/monitor/api.lua:10: in main chunk
	[C]: in function 'require'
	./orange/orange.lua:220: in function 'init_worker'
	init_worker_by_lua:3: in main chunk
2016/09/21 16:54:41 [error] 5082#0: init_worker_by_lua error: ./orange/plugins/monitor/api.lua:10: module 'orange.plugins.monitor_db' not found:
	no field package.preload['orange.plugins.monitor_db']
	no file '../orange/plugins/monitor_db.lua'
	no file './lor/lor/orange/plugins/monitor_db.lua'
	no file './lor/lor/resty/orange/plugins/monitor_db.lua'
	no file '/opt/openresty/lualib/orange/plugins/monitor_db.lua'
	no file '/opt/openresty/lualib/orange/plugins/monitor_db/init.lua'
	no file './orange/plugins/monitor_db.lua'
	no file '/opt/openresty/luajit/share/luajit-2.1.0-beta1/orange/plugins/monitor_db.lua'
	no file '/usr/local/share/lua/5.1/orange/plugins/monitor_db.lua'
	no file '/usr/local/share/lua/5.1/orange/plugins/monitor_db/init.lua'
	no file '/opt/openresty/luajit/share/lua/5.1/orange/plugins/monitor_db.lua'
	no file '/opt/openresty/luajit/share/lua/5.1/orange/plugins/monitor_db/init.lua'
	no file '/opt/openresty/lualib/orange/plugins/monitor_db.so'
	no file './orange/plugins/monitor_db.so'
	no file '/usr/local/lib/lua/5.1/orange/plugins/monitor_db.so'
	no file '/opt/openresty/luajit/lib/lua/5.1/orange/plugins/monitor_db.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
	no file '/opt/openresty/lualib/orange.so'
	no file './orange.so'
	no file '/usr/local/lib/lua/5.1/orange.so'
	no file '/opt/openresty/luajit/lib/lua/5.1/orange.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
stack traceback:
	[C]: in function 'require'
	./orange/plugins/monitor/api.lua:10: in main chunk
	[C]: in function 'require'
	./orange/orange.lua:220: in function 'init_worker'
	init_worker_by_lua:3: in main chunk
2016/09/21 16:54:41 [error] 5079#0: init_worker_by_lua error: ./orange/plugins/monitor/api.lua:10: module 'orange.plugins.monitor_db' not found:
	no field package.preload['orange.plugins.monitor_db']
	no file '../orange/plugins/monitor_db.lua'
	no file './lor/lor/orange/plugins/monitor_db.lua'
	no file './lor/lor/resty/orange/plugins/monitor_db.lua'
	no file '/opt/openresty/lualib/orange/plugins/monitor_db.lua'
	no file '/opt/openresty/lualib/orange/plugins/monitor_db/init.lua'
	no file './orange/plugins/monitor_db.lua'
	no file '/opt/openresty/luajit/share/luajit-2.1.0-beta1/orange/plugins/monitor_db.lua'
	no file '/usr/local/share/lua/5.1/orange/plugins/monitor_db.lua'
	no file '/usr/local/share/lua/5.1/orange/plugins/monitor_db/init.lua'
	no file '/opt/openresty/luajit/share/lua/5.1/orange/plugins/monitor_db.lua'
	no file '/opt/openresty/luajit/share/lua/5.1/orange/plugins/monitor_db/init.lua'
	no file '/opt/openresty/lualib/orange/plugins/monitor_db.so'
	no file './orange/plugins/monitor_db.so'
	no file '/usr/local/lib/lua/5.1/orange/plugins/monitor_db.so'
	no file '/opt/openresty/luajit/lib/lua/5.1/orange/plugins/monitor_db.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
	no file '/opt/openresty/lualib/orange.so'
	no file './orange.so'
	no file '/usr/local/lib/lua/5.1/orange.so'
	no file '/opt/openresty/luajit/lib/lua/5.1/orange.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
stack traceback:
	[C]: in function 'require'
	./orange/plugins/monitor/api.lua:10: in main chunk
	[C]: in function 'require'
	./orange/orange.lua:220: in function 'init_worker'
	init_worker_by_lua:3: in main chunk
2016/09/21 16:55:42 [notice] 5130#0: signal process started
ngx_http_fastdfs_process_exit pid=5078
ngx_http_fastdfs_process_exit pid=5079
ngx_http_fastdfs_process_exit pid=5080
ngx_http_fastdfs_process_exit pid=5082
2016/09/21 16:55:42 [error] 5133#0: init_by_lua error: ./orange/plugins/monitor/api.lua:10: module 'orange.plugins.monitor_db' not found:
	no field package.preload['orange.plugins.monitor_db']
	no file '../orange/plugins/monitor_db.lua'
	no file './lor/lor/orange/plugins/monitor_db.lua'
	no file './lor/lor/resty/orange/plugins/monitor_db.lua'
	no file '/opt/openresty/lualib/orange/plugins/monitor_db.lua'
	no file '/opt/openresty/lualib/orange/plugins/monitor_db/init.lua'
	no file './orange/plugins/monitor_db.lua'
	no file '/opt/openresty/luajit/share/luajit-2.1.0-beta1/orange/plugins/monitor_db.lua'
	no file '/usr/local/share/lua/5.1/orange/plugins/monitor_db.lua'
	no file '/usr/local/share/lua/5.1/orange/plugins/monitor_db/init.lua'
	no file '/opt/openresty/luajit/share/lua/5.1/orange/plugins/monitor_db.lua'
	no file '/opt/openresty/luajit/share/lua/5.1/orange/plugins/monitor_db/init.lua'
	no file '/opt/openresty/lualib/orange/plugins/monitor_db.so'
	no file './orange/plugins/monitor_db.so'
	no file '/usr/local/lib/lua/5.1/orange/plugins/monitor_db.so'
	no file '/opt/openresty/luajit/lib/lua/5.1/orange/plugins/monitor_db.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
	no file '/opt/openresty/lualib/orange.so'
	no file './orange.so'
	no file '/usr/local/lib/lua/5.1/orange.so'
	no file '/opt/openresty/luajit/lib/lua/5.1/orange.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
stack traceback:
	[C]: in function 'require'
	./orange/plugins/monitor/api.lua:10: in main chunk
	[C]: in function 'require'
	./orange/orange.lua:13: in main chunk
	[C]: in function 'require'
	init_by_lua:3: in main chunk
ngx_http_fastdfs_process_init pid=5168
ngx_http_fastdfs_process_init pid=5166
ngx_http_fastdfs_process_init pid=5165
ngx_http_fastdfs_process_init pid=5164
[2016-09-21 16:59:03] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 16:59:03] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 16:59:03] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 16:59:03] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 16:59:03] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 16:59:03] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 16:59:03] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 16:59:03] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
2016/09/21 17:50:24 [notice] 5502#0: signal process started
ngx_http_fastdfs_process_exit pid=5164
ngx_http_fastdfs_process_exit pid=5165
ngx_http_fastdfs_process_exit pid=5166
ngx_http_fastdfs_process_exit pid=5168
ngx_http_fastdfs_process_init pid=5510
ngx_http_fastdfs_process_init pid=5509
ngx_http_fastdfs_process_init pid=5508
ngx_http_fastdfs_process_init pid=5507
[2016-09-21 17:50:33] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 17:50:33] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 17:50:33] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 17:50:33] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 17:50:33] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 17:50:33] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 17:50:33] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 17:50:33] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
2016/09/21 18:15:10 [notice] 5786#0: signal process started
ngx_http_fastdfs_process_exit pid=5507
ngx_http_fastdfs_process_exit pid=5508
ngx_http_fastdfs_process_exit pid=5509
ngx_http_fastdfs_process_exit pid=5510
ngx_http_fastdfs_process_init pid=5791
ngx_http_fastdfs_process_init pid=5795
ngx_http_fastdfs_process_init pid=5794
ngx_http_fastdfs_process_init pid=5793
[2016-09-21 18:15:19] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 18:15:19] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 18:15:19] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 18:15:19] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 18:15:19] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 18:15:19] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 18:15:19] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 18:15:19] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
2016/09/21 18:15:40 [info] 5794#0: *2 epoll_wait() reported that client prematurely closed connection, so upstream connection is closed too while connecting to upstream, client: 192.168.224.1, server: , request: "GET /hshimages/13lc.png HTTP/1.1", upstream: "http://121.43.165.241:6688/hshimages/13lc.png", host: "192.168.224.128:8088", referrer: "http://192.168.224.128:8088/index.html"
2016/09/21 18:15:40 [info] 5794#0: *14 epoll_wait() reported that client prematurely closed connection, so upstream connection is closed too while reading upstream, client: 192.168.224.1, server: , request: "GET /hshimages/foot.jpg HTTP/1.1", upstream: "http://121.43.165.241:6688/hshimages/foot.jpg", host: "192.168.224.128:8088", referrer: "http://192.168.224.128:8088/index.html"
2016/09/21 18:15:40 [info] 5794#0: *5 epoll_wait() reported that client prematurely closed connection, so upstream connection is closed too while reading upstream, client: 192.168.224.1, server: , request: "GET /hshimages/%E9%A6%96%E9%A1%B5%E6%A8%AA%E5%B9%85.jpg HTTP/1.1", upstream: "http://121.43.165.241:6688/hshimages/%E9%A6%96%E9%A1%B5%E6%A8%AA%E5%B9%85.jpg", host: "192.168.224.128:8088", referrer: "http://192.168.224.128:8088/index.html"
2016/09/21 18:15:40 [info] 5794#0: *4 epoll_wait() reported that client prematurely closed connection, so upstream connection is closed too while reading upstream, client: 192.168.224.1, server: , request: "GET /hshimages/hsh1.png HTTP/1.1", upstream: "http://121.43.165.241:6688/hshimages/hsh1.png", host: "192.168.224.128:8088", referrer: "http://192.168.224.128:8088/index.html"
2016/09/21 18:15:40 [info] 5794#0: *3 epoll_wait() reported that client prematurely closed connection, so upstream connection is closed too while sending request to upstream, client: 192.168.224.1, server: , request: "GET /hshimages/09lc.png HTTP/1.1", upstream: "http://121.43.165.241:6688/hshimages/09lc.png", host: "192.168.224.128:8088", referrer: "http://192.168.224.128:8088/index.html"
2016/09/21 18:33:09 [info] 5791#0: *92 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/21 18:33:09 [info] 5791#0: *93 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/21 18:33:09 [info] 5791#0: *94 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/21 18:33:09 [info] 5791#0: *95 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/21 18:33:09 [info] 5791#0: *96 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/21 18:33:09 [info] 5791#0: *97 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/21 18:36:47 [info] 5794#0: *197 [lua] handler.lua:74: access(): [WAF-Forbidden-Rule] http://192.168.224.128:8088/index.html uri:/index.html, client: 192.168.224.1, server: , request: "GET /index.html HTTP/1.1", host: "192.168.224.128:8088"
2016/09/21 18:56:32 [notice] 6091#0: signal process started
ngx_http_fastdfs_process_exit pid=5791
ngx_http_fastdfs_process_exit pid=5793
ngx_http_fastdfs_process_exit pid=5794
ngx_http_fastdfs_process_exit pid=5795
ngx_http_fastdfs_process_init pid=6100
ngx_http_fastdfs_process_init pid=6098
ngx_http_fastdfs_process_init pid=6097
ngx_http_fastdfs_process_init pid=6096
[2016-09-21 18:56:41] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 18:56:41] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 18:56:41] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 18:56:41] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 18:56:41] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 18:56:41] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 18:56:41] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 18:56:41] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
2016/09/21 18:56:41 [error] 6096#0: lua entry thread aborted: runtime error: ./orange/orange.lua:171: Expected object key string but found invalid token at character 2
stack traceback:
coroutine 0:
	[C]: in function 'decode'
	./orange/orange.lua:171: in function 'load_data_by_pgsql'
	./orange/orange.lua:230: in function <./orange/orange.lua:224>, context: ngx.timer
2016/09/21 18:59:19 [info] 6096#0: *95 epoll_wait() reported that client prematurely closed connection, so upstream connection is closed too while sending request to upstream, client: 192.168.224.1, server: , request: "GET /hshimages/LOGO.png HTTP/1.1", upstream: "http://121.43.165.241:6688/hshimages/LOGO.png", host: "192.168.224.128:8088", referrer: "http://192.168.224.128:8088/index.html"
2016/09/21 19:00:00 [info] 6096#0: *94 epoll_wait() reported that client prematurely closed connection, so upstream connection is closed too while sending request to upstream, client: 192.168.224.1, server: , request: "GET /hshimages/LOGO.png HTTP/1.1", upstream: "http://121.43.165.241:6688/hshimages/LOGO.png", host: "192.168.224.128:8088", referrer: "http://192.168.224.128:8088/index.html"
2016/09/21 22:30:07 [notice] 7259#0: signal process started
ngx_http_fastdfs_process_exit pid=6096
ngx_http_fastdfs_process_exit pid=6098
ngx_http_fastdfs_process_exit pid=6100
ngx_http_fastdfs_process_exit pid=6097
ngx_http_fastdfs_process_init pid=7264
ngx_http_fastdfs_process_init pid=7266
ngx_http_fastdfs_process_init pid=7267
ngx_http_fastdfs_process_init pid=7268
[2016-09-21 22:30:16] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 22:30:16] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 22:30:16] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 22:30:16] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 22:30:16] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 22:30:16] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-21 22:30:16] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-21 22:30:16] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
2016/09/21 22:30:16 [error] 7264#0: lua entry thread aborted: runtime error: ./orange/orange.lua:171: Expected object key string but found invalid token at character 2
stack traceback:
coroutine 0:
	[C]: in function 'decode'
	./orange/orange.lua:171: in function 'load_data_by_pgsql'
	./orange/orange.lua:230: in function <./orange/orange.lua:224>, context: ngx.timer
2016/09/22 10:38:25 [notice] 1818#0: signal process started
2016/09/22 10:38:25 [alert] 1818#0: kill(7263, 15) failed (3: No such process)
ngx_http_fastdfs_process_init pid=1827
ngx_http_fastdfs_process_init pid=1825
ngx_http_fastdfs_process_init pid=1823
ngx_http_fastdfs_process_init pid=1824
[2016-09-22 10:38:34] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 10:38:34] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 10:38:34] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 10:38:34] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 10:38:34] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 10:38:34] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 10:38:34] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 10:38:34] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
2016/09/22 10:38:34 [error] 1823#0: lua entry thread aborted: runtime error: ./orange/orange.lua:171: Expected object key string but found invalid token at character 2
stack traceback:
coroutine 0:
	[C]: in function 'decode'
	./orange/orange.lua:171: in function 'load_data_by_pgsql'
	./orange/orange.lua:230: in function <./orange/orange.lua:224>, context: ngx.timer
2016/09/22 10:47:53 [info] 1823#0: *11 [lua] handler.lua:100: redirect(): [Redirect] / to:http://192.168.224.128:8088/index.html, client: 192.168.224.1, server: , request: "GET / HTTP/1.1", host: "192.168.224.128:8088"
2016/09/22 10:48:15 [info] 1823#0: *12 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 10:48:15 [info] 1823#0: *13 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 10:48:15 [info] 1823#0: *14 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 10:48:15 [info] 1823#0: *15 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 10:48:15 [info] 1823#0: *16 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 10:48:35 [info] 1823#0: *19 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 10:48:35 [info] 1823#0: *20 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 10:48:35 [info] 1823#0: *21 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 10:48:35 [info] 1823#0: *22 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 10:48:35 [info] 1823#0: *23 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 10:49:02 [error] 1823#0: *24 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 192.168.224.1, server: , request: "GET /hshimages/HSH4.png HTTP/1.1", upstream: "http://121.43.165.241:6688/hshimages/HSH4.png", host: "192.168.224.128:8088", referrer: "http://192.168.224.128:8088/index.html"
2016/09/22 10:49:02 [notice] 2143#0: signal process started
ngx_http_fastdfs_process_exit pid=1823
ngx_http_fastdfs_process_exit pid=1827
ngx_http_fastdfs_process_exit pid=1825
ngx_http_fastdfs_process_exit pid=1824
ngx_http_fastdfs_process_init pid=2152
ngx_http_fastdfs_process_init pid=2150
ngx_http_fastdfs_process_init pid=2149
ngx_http_fastdfs_process_init pid=2148
[2016-09-22 10:49:12] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 10:49:12] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 10:49:12] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 10:49:12] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 10:49:12] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 10:49:12] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 10:49:12] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 10:49:12] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
2016/09/22 10:49:12 [error] 2148#0: lua entry thread aborted: runtime error: ./orange/orange.lua:171: Expected object key string but found invalid token at character 2
stack traceback:
coroutine 0:
	[C]: in function 'decode'
	./orange/orange.lua:171: in function 'load_data_by_pgsql'
	./orange/orange.lua:230: in function <./orange/orange.lua:224>, context: ngx.timer
2016/09/22 10:49:47 [info] 2150#0: *9 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 10:49:47 [info] 2150#0: *10 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 10:49:47 [info] 2150#0: *11 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 10:49:47 [info] 2150#0: *12 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 10:49:47 [info] 2150#0: *13 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 10:50:05 [info] 2148#0: *15 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 10:50:05 [info] 2148#0: *16 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 10:50:05 [info] 2148#0: *17 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 10:50:05 [info] 2148#0: *18 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 10:50:05 [info] 2148#0: *19 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 10:50:11 [info] 2150#0: *8 [lua] handler.lua:100: redirect(): [Redirect] /i to:http://192.168.224.128:8088/index.html, client: 192.168.224.1, server: , request: "GET /i HTTP/1.1", host: "192.168.224.128:8088"
2016/09/22 10:51:11 [info] 2148#0: *21 client timed out (110: Connection timed out) while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 10:51:11 [info] 2148#0: *22 client timed out (110: Connection timed out) while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 10:51:11 [info] 2148#0: *23 client timed out (110: Connection timed out) while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 10:51:11 [info] 2148#0: *24 client timed out (110: Connection timed out) while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 10:51:11 [info] 2148#0: *25 client timed out (110: Connection timed out) while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 10:55:04 [notice] 2338#0: signal process started
ngx_http_fastdfs_process_exit pid=2148
ngx_http_fastdfs_process_exit pid=2149
ngx_http_fastdfs_process_exit pid=2150
ngx_http_fastdfs_process_exit pid=2152
ngx_http_fastdfs_process_init pid=2346
ngx_http_fastdfs_process_init pid=2345
ngx_http_fastdfs_process_init pid=2344
ngx_http_fastdfs_process_init pid=2343
[2016-09-22 10:55:13] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 10:55:13] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 10:55:13] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 10:55:13] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 10:55:13] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 10:55:13] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 10:55:13] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 10:55:13] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
2016/09/22 10:55:13 [error] 2343#0: lua entry thread aborted: runtime error: ./orange/orange.lua:171: Expected object key string but found invalid token at character 2
stack traceback:
coroutine 0:
	[C]: in function 'decode'
	./orange/orange.lua:171: in function 'load_data_by_pgsql'
	./orange/orange.lua:230: in function <./orange/orange.lua:224>, context: ngx.timer
2016/09/22 10:56:37 [error] 2345#0: *13 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 192.168.224.1, server: , request: "GET /hshimages/lc1on.jpg HTTP/1.1", upstream: "http://121.43.165.241:6688/hshimages/lc1on.jpg", host: "192.168.224.128:8088", referrer: "http://192.168.224.128:8088/index.html"
ngx_http_fastdfs_process_exit pid=2344
ngx_http_fastdfs_process_exit pid=2345
ngx_http_fastdfs_process_exit pid=2346
ngx_http_fastdfs_process_exit pid=2343
ngx_http_fastdfs_process_init pid=2548
ngx_http_fastdfs_process_init pid=2546
ngx_http_fastdfs_process_init pid=2545
ngx_http_fastdfs_process_init pid=2544
[2016-09-22 11:10:42] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 11:10:42] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 11:10:42] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 11:10:42] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 11:10:42] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 11:10:42] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 11:10:42] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 11:10:42] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
2016/09/22 11:10:42 [error] 2544#0: lua entry thread aborted: runtime error: ./orange/orange.lua:171: Expected object key string but found invalid token at character 2
stack traceback:
coroutine 0:
	[C]: in function 'decode'
	./orange/orange.lua:171: in function 'load_data_by_pgsql'
	./orange/orange.lua:230: in function <./orange/orange.lua:224>, context: ngx.timer
2016/09/22 12:53:49 [notice] 3142#0: signal process started
ngx_http_fastdfs_process_exit pid=2544
ngx_http_fastdfs_process_exit pid=2545
ngx_http_fastdfs_process_exit pid=2546
ngx_http_fastdfs_process_exit pid=2548
ngx_http_fastdfs_process_init pid=3147
ngx_http_fastdfs_process_init pid=3149
ngx_http_fastdfs_process_init pid=3150
ngx_http_fastdfs_process_init pid=3151
[2016-09-22 12:53:58] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 12:53:58] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 12:53:58] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 12:53:58] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 12:53:58] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 12:53:58] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 12:53:58] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 12:53:58] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
2016/09/22 12:54:37 [notice] 3231#0: signal process started
ngx_http_fastdfs_process_exit pid=3147
ngx_http_fastdfs_process_exit pid=3149
ngx_http_fastdfs_process_exit pid=3150
ngx_http_fastdfs_process_exit pid=3151
ngx_http_fastdfs_process_init pid=3240
ngx_http_fastdfs_process_init pid=3237
ngx_http_fastdfs_process_init pid=3238
ngx_http_fastdfs_process_init pid=3236
[2016-09-22 12:54:46] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 12:54:46] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 12:54:46] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 12:54:46] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 12:54:46] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 12:54:46] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 12:54:46] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 12:54:46] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
2016/09/22 12:55:40 [info] 3237#0: *4 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 12:55:40 [info] 3237#0: *5 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 12:55:40 [info] 3237#0: *6 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 12:55:40 [info] 3237#0: *7 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 12:55:40 [info] 3237#0: *8 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 12:55:58 [info] 3237#0: *10 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 12:55:58 [info] 3237#0: *11 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 12:55:58 [info] 3237#0: *12 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 12:55:58 [info] 3237#0: *13 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 12:55:58 [info] 3237#0: *14 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 12:57:12 [info] 3237#0: *18 client timed out (110: Connection timed out) while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 12:57:12 [info] 3237#0: *19 client timed out (110: Connection timed out) while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 12:57:12 [info] 3237#0: *20 client timed out (110: Connection timed out) while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 12:57:12 [info] 3237#0: *21 client timed out (110: Connection timed out) while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 12:57:12 [info] 3237#0: *22 client timed out (110: Connection timed out) while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 12:58:35 [notice] 3390#0: signal process started
ngx_http_fastdfs_process_exit pid=3236
ngx_http_fastdfs_process_exit pid=3237
ngx_http_fastdfs_process_exit pid=3238
ngx_http_fastdfs_process_exit pid=3240
ngx_http_fastdfs_process_init pid=3395
ngx_http_fastdfs_process_init pid=3396
ngx_http_fastdfs_process_init pid=3398
ngx_http_fastdfs_process_init pid=3399
[2016-09-22 12:58:44] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 12:58:44] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 12:58:44] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 12:58:44] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 12:58:44] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 12:58:44] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 12:58:44] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 12:58:44] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
2016/09/22 12:59:50 [notice] 3451#0: signal process started
ngx_http_fastdfs_process_exit pid=3395
ngx_http_fastdfs_process_exit pid=3396
ngx_http_fastdfs_process_exit pid=3398
ngx_http_fastdfs_process_exit pid=3399
ngx_http_fastdfs_process_init pid=3460
ngx_http_fastdfs_process_init pid=3458
ngx_http_fastdfs_process_init pid=3457
ngx_http_fastdfs_process_init pid=3456
[2016-09-22 12:59:59] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 12:59:59] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 12:59:59] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 12:59:59] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 12:59:59] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 12:59:59] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 12:59:59] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 12:59:59] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
2016/09/22 13:02:39 [info] 3460#0: *3 client timed out (110: Connection timed out) while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:02:39 [info] 3460#0: *4 client timed out (110: Connection timed out) while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:02:39 [info] 3460#0: *5 client timed out (110: Connection timed out) while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:02:39 [info] 3460#0: *6 client timed out (110: Connection timed out) while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:02:39 [info] 3460#0: *8 client timed out (110: Connection timed out) while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
ngx_http_fastdfs_process_exit pid=3456
ngx_http_fastdfs_process_exit pid=3457
ngx_http_fastdfs_process_exit pid=3460
ngx_http_fastdfs_process_exit pid=3458
ngx_http_fastdfs_process_init pid=3588
ngx_http_fastdfs_process_init pid=3586
ngx_http_fastdfs_process_init pid=3585
ngx_http_fastdfs_process_init pid=3584
[2016-09-22 13:03:04] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 13:03:04] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 13:03:04] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 13:03:04] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 13:03:04] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 13:03:04] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 13:03:04] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 13:03:04] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
2016/09/22 13:03:22 [info] 3585#0: *6 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:03:22 [info] 3585#0: *7 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:03:22 [info] 3585#0: *8 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:04:16 [info] 3588#0: *9 client timed out (110: Connection timed out) while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:04:16 [info] 3588#0: *10 client timed out (110: Connection timed out) while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:04:16 [info] 3588#0: *11 client timed out (110: Connection timed out) while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:04:22 [info] 3588#0: *12 client timed out (110: Connection timed out) while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:04:22 [info] 3588#0: *13 client timed out (110: Connection timed out) while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:04:22 [info] 3588#0: *14 client timed out (110: Connection timed out) while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:08:03 [notice] 3675#0: signal process started
ngx_http_fastdfs_process_exit pid=3584
ngx_http_fastdfs_process_exit pid=3588
ngx_http_fastdfs_process_exit pid=3586
ngx_http_fastdfs_process_exit pid=3585
ngx_http_fastdfs_process_init pid=3684
ngx_http_fastdfs_process_init pid=3682
ngx_http_fastdfs_process_init pid=3681
ngx_http_fastdfs_process_init pid=3680
[2016-09-22 13:08:12] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 13:08:12] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 13:08:12] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 13:08:12] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 13:08:12] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 13:08:12] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 13:08:12] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 13:08:12] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
2016/09/22 13:08:56 [info] 3684#0: *6 [lua] handler.lua:100: redirect(): [Redirect] /in to:http://192.168.224.128:8088/index.html, client: 192.168.224.1, server: , request: "GET /in HTTP/1.1", host: "192.168.224.128:8088"
2016/09/22 13:09:08 [info] 3684#0: *7 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:09:08 [info] 3684#0: *8 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:09:08 [info] 3684#0: *9 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:09:08 [info] 3684#0: *10 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:10:15 [info] 3684#0: *12 client timed out (110: Connection timed out) while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:10:15 [info] 3684#0: *13 client timed out (110: Connection timed out) while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:10:15 [info] 3684#0: *14 client timed out (110: Connection timed out) while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:10:20 [info] 3684#0: *15 client timed out (110: Connection timed out) while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:10:20 [info] 3684#0: *16 client timed out (110: Connection timed out) while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:18:48 [info] 3684#0: *23 client 192.168.224.1 closed keepalive connection (104: Connection reset by peer)
2016/09/22 13:19:01 [info] 3684#0: *17 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:19:01 [info] 3684#0: *18 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:19:01 [info] 3684#0: *19 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:19:01 [info] 3684#0: *20 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:19:01 [info] 3684#0: *21 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:19:01 [info] 3684#0: *22 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:20:09 [info] 3684#0: *26 client timed out (110: Connection timed out) while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:20:09 [info] 3684#0: *27 client timed out (110: Connection timed out) while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:20:09 [info] 3684#0: *28 client timed out (110: Connection timed out) while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:20:09 [info] 3684#0: *29 client timed out (110: Connection timed out) while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:20:09 [info] 3684#0: *30 client timed out (110: Connection timed out) while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:21:09 [info] 3684#0: *33 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:21:09 [info] 3684#0: *34 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:21:09 [info] 3684#0: *35 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:21:09 [info] 3684#0: *36 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:21:09 [info] 3684#0: *37 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:21:46 [info] 3684#0: *39 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:21:46 [info] 3684#0: *40 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:21:46 [info] 3684#0: *41 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:21:46 [info] 3684#0: *42 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 13:21:46 [info] 3684#0: *43 client closed connection while waiting for request, client: 192.168.224.1, server: 0.0.0.0:8088
2016/09/22 14:47:46 [notice] 4173#0: signal process started
ngx_http_fastdfs_process_exit pid=3684
ngx_http_fastdfs_process_exit pid=3680
ngx_http_fastdfs_process_exit pid=3681
ngx_http_fastdfs_process_exit pid=3682
ngx_http_fastdfs_process_init pid=4178
ngx_http_fastdfs_process_init pid=4182
ngx_http_fastdfs_process_init pid=4181
ngx_http_fastdfs_process_init pid=4180
[2016-09-22 14:47:55] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 14:47:55] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 14:47:55] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 14:47:55] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 14:47:55] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 14:47:55] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 14:47:55] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 14:47:55] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
2016/09/22 14:53:05 [notice] 4328#0: signal process started
ngx_http_fastdfs_process_exit pid=4182
ngx_http_fastdfs_process_exit pid=4178
ngx_http_fastdfs_process_exit pid=4180
ngx_http_fastdfs_process_exit pid=4181
ngx_http_fastdfs_process_init pid=4337
ngx_http_fastdfs_process_init pid=4335
ngx_http_fastdfs_process_init pid=4334
ngx_http_fastdfs_process_init pid=4333
[2016-09-22 14:53:14] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 14:53:14] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 14:53:14] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 14:53:14] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 14:53:14] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 14:53:14] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
[2016-09-22 14:53:14] ERROR - file: ../tracker/tracker_proto.c, line: 590, connect to tracker server 192.168.0.102:22122 fail, errno: 110, error info: Connection timed out
[2016-09-22 14:53:14] INFO - fastdfs apache / nginx module v1.15, response_mode=proxy, base_path=/home/fengz/fastdfs_ngx, url_have_group_name=0, group_name=group1, storage_server_port=23000, path_count=1, store_path0=/home/fengz/fastdfs_storage_data, connect_timeout=2, network_timeout=30, tracker_server_count=1, if_alias_prefix=, local_host_ip_count=2, anti_steal_token=0, token_ttl=0s, anti_steal_secret_key length=0, token_check_fail content_type=, token_check_fail buff length=0, load_fdfs_parameters_from_tracker=1, storage_sync_file_max_delay=86400s, use_storage_id=0, storage server id count=0, flv_support=1, flv_extension=flv
